### PR TITLE
Update Sport Store using Flexbox, Grid, Float for DIV

### PR DESCRIPTION
**Pull request by ledung1974**
In my branch "Le_Dung", I have already upload "Sport-store" Web design. Can I merge this folder to master's branch?

### CODE REVIEW

File: applications/sport-store/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            <table>
+              <tr>
+                <th>Sport Items</th>
+                <th>Price</th>
+              </tr>
+              <tr>
+                <td>
+                  <head>
+                    Men's Terrex Agravic Flow Shoes
+                  </head>
```
Comment (tupes) [inline applications/sport-store/index.html]: You shouldn't be using a `<head>` element here.

**Review by tupes — CHANGES_REQUESTED** [review]
Great job!

Just that one small fix and I'll approve this. It might be kinda confusing that we'll have two sports e-commerce applications, but I'll explain that to the class.

File: applications/sport-store/index.html (hunk 1 above)
Comment (ledung1974) [reply applications/sport-store/index.html]: Oh ! I didn't know why <head> element is occurred in this code. Thanks !

**Comment by ledung1974** [comment]
I know that "Responsive Web Design" is important. Next time I will focus on this.

**Review by tupes — APPROVED** [review]
Great job!

There might be a few minor changes I'd recommend, but we can discuss in class.

**Pull request merged**